index.js

```diff
@@ -1,9 +1,16 @@
-function moveToMapPosition(referenceMap, mapToMove) {
-  mapToMove.jumpTo({
-    center: referenceMap.getCenter(),
-    zoom: referenceMap.getZoom(),
-    bearing: referenceMap.getBearing(),
-    pitch: referenceMap.getPitch()
+function moveToMapPosition (master, slaves) {
+  var center = master.getCenter();
+  var zoom = master.getZoom();
+  var bearing = master.getBearing();
+  var pitch = master.getPitch();
+
+  slaves.forEach(slave => {
```

Comment:
Please use ES5 syntax here, since this module is not transpiled and probably shouldn't rely on arrow functions unless absolutely necessary.